### ISSUE TITLE
load multiple files via arguments

### ISSUE THREAD

**Issue by athompson673** [JLouis-B/RedTools#5]
New feature request: provide multiple file paths as an argument, and attempt to load each one using `loadFileGeneric`.

I don't have an environment to build / test this, but it seems like this could be a simple edit to main.cpp:

```
#include "GUI_MainWindow.h"
#include <QApplication>


int main(int argc, char *argv[])
{
    QApplication a(argc, argv);

    GUI_MainWindow w;
    w.show();
    w.initIrrlicht();

    int n_arguments = QCoreApplication::arguments().size()
    if (n_arguments > 1)
    {
        for (int i = 1; i < n_arguments; i++)
            w.loadFileGeneric(QCoreApplication::arguments().at(i));
    }
    
    return a.exec();
}
```

**Comment by JLouis-B** [comment]
Hi,
Thanks for the suggestion, it seems to be a good idea indeed.
However it should be noted that currently the loading of several meshes in the software is limited and goes through the "Add mesh" button which actually merges the extra mesh to the first one.
I would like to have a real scene management in the future, but I can still implement your suggestion for the next version, it won't be extremely complicated.

**Comment by JLouis-B** [comment]
Implemented in the 2.13 release
